settings.gradle

```diff
@@ -2,4 +2,5 @@ rootProject.name = "Kevin"
 include ':demo'
 include ':core'
 include ':accounts'
-include ':in-app-payments'
\ No newline at end of file
+include ':in-app-payments'
+include ':testcore'
```

Comment:
are you sure about having this in a separate module? maybe just putting all these stuff into core module would be sufficient?